cedar-policy-cli/src/lib.rs

```diff
@@ -114,6 +132,9 @@ pub struct ValidateArgs {
     /// experimental `partial-validate` feature enabled.
     #[arg(long = "partial-validate")]
     pub partial_validate: bool,
+    /// Schema format (JSON or Custom)
+    #[arg(long, value_enum, default_value_t = SchemaFormat::Json)]
```

Comment:
We should change the default to be non-JSON, and then explicitly use it in our existing codebase. We want users to think of the custom format as the preferred one.

Reply:
Is that a breaking change?

Reply:
That's a breaking change we should make for Cedar 4.  But we shouldn't make that breaking change in this PR, because we intend this PR for Cedar 3.1